pyi.Dockerfile

```diff
@@ -111,33 +111,72 @@ RUN ./node_builder_musl_env.sh
 COPY scripts/build_node_package.sh .
 RUN ./build_node_package.sh
 
-# bcc helpers
+# building bcc along with helpers
 # built on newer Ubuntu because they require new clang (newer than available in GPROFILER_BUILDER's CentOS 7)
 # these are only relevant for modern kernels, so there's no real reason to build them on CentOS 7 anyway.
-FROM ubuntu${PYPERF_BUILDER_UBUNTU} AS bcc-helpers
+FROM ubuntu${PYPERF_BUILDER_UBUNTU} AS bcc-build
 WORKDIR /tmp
 
 RUN if [ "$(uname -m)" = "aarch64" ]; then \
         exit 0; \
     fi && \
     apt-get update && \
-    apt-get install -y --no-install-recommends \
-        clang-10 \
-        libelf-dev \
-        make \
-        build-essential \
-        llvm \
-        ca-certificates \
-        git
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+      curl \
+      build-essential \
+      iperf llvm-12-dev \
+      clang-12 libclang-12-dev \
+      cmake \
+      python3 python3-pip \
+      flex \
+      libfl-dev \
+      bison \
+      libelf-dev \
+      libz-dev \
+      liblzma-dev \
+      ca-certificates \
+      git \
+      patchelf scons
 
 COPY --from=perf-builder /bpftool /bpftool
 
 COPY scripts/bcc_helpers_build.sh .
 RUN ./bcc_helpers_build.sh
 
+# build bcc
+# TODO: copied from the main Dockerfile... but modified a lot. we'd want to share it some day.
```

Comment:
Btw - now that BCC & BCC helpers are built both on Ubuntu, can we share more of their build? Only if easy.

Reply:
👍 
That's a good idea, I guess it should work.
I'll look into it.

Reply:
I mean between the 2 dockerfiles - both now use the same base - `FROM ubuntu${PYPERF_BUILDER_UBUNTU} AS bcc-builder-base` in Dockerfile and `FROM ubuntu${PYPERF_BUILDER_UBUNTU} AS bcc-build` in pyi.Dockerfile, so they can be shared with eg `bcc_env.sh` and `bcc_build.sh` scripts like the rest of the artifacts that we build.

Reply:
👍